frontend/src/pages/modelServing/screens/global/ModelServingProjectSelection.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Bullseye, Split, SplitItem } from '@patternfly/react-core';
+import ProjectSelectorNavigator from '~/concepts/projects/ProjectSelectorNavigator';
+import { KnownLabels } from '~/k8sTypes';
+
+type ModelServingProjectSelectionProps = {
+  getRedirectPath: (namespace: string) => string;
+};
+
+const ModelServingProjectSelection: React.FC<ModelServingProjectSelectionProps> = ({
+  getRedirectPath,
+}) => (
+  <Split hasGutter>
+    <SplitItem>
+      <Bullseye>Project</Bullseye>
+    </SplitItem>
+    <SplitItem>
+      {/* Maybe we want to filter the projects with no deployed models that's why I added the filterLable prop */}
+      <ProjectSelectorNavigator
+        getRedirectPath={getRedirectPath}
+        invalidDropdownPlaceholder={'All projects'}
+        selectAllProjects
+        filterLabel={KnownLabels.DASHBOARD_RESOURCE}
```

Comment:
Is this necessary? Why would we ever want to filter this list down?

Reply:
Will be fixed in https://github.com/opendatahub-io/odh-dashboard/pull/2154